AODN/AODN-WAVE-DM/BOM-WAVE-DM/bom_wave_library/generate_nc_file_att

```diff
@@ -67,6 +66,5 @@ units = degrees_east
 axis = X
 valid_min = -180.0
 valid_max = 180.0
-_FillValue = 
```

Comment:
the attribute fillValue is required. Is it added somewhere else in the script?

Reply:
yes, it adds it somewhere else, all goog